app/routes.py

```diff
@@ -1,2 +1,297 @@
-from flask import Blueprint
+
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response, jsonify
+import datetime
+import os 
+import requests
+
+goal_bp = task_bp = Blueprint("goal", __name__, url_prefix="/goals")
```

Comment:
*Small Suggestion*:  I suggest defining both the goal_bp and task_bp together at the top, just so you can see the list of blueprints available at the top.  Small readability suggestion.